grid_map_common/cmake/grid_map_package.cmake

```diff
@@ -0,0 +1,42 @@
+# Standard grid_map project setup
+#
+# @public
+#
+macro(grid_map_package)
+  if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
+    message(STATUS "Setting build type to Release as none was specified.")
+    set(CMAKE_BUILD_TYPE "Release" CACHE
+        STRING "Choose the type of build." FORCE)
+    # Set the possible values of build type for cmake-gui
+    set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS
+      "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
+  endif()
+
+  # Default to C++14
+  if(NOT CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 14)
+  endif()
+
+  if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+    add_compile_options(-Wall -Wextra -Wpedantic -Werror -Wdeprecated -fPIC)
+  endif()
+
+  option(COVERAGE_ENABLED "Enable code coverage" FALSE)
```

Comment:
I don't think they do coverage metrics here - this section can be removed. I think the rest are fine.